test/unit/routes/accessibility-statement.test.js

```diff
@@ -0,0 +1,27 @@
+import { startServer } from '../../../src/utils/start-server.js'
```

Comment:
Can we move this into `test/integration` please? 

There we've been tending to use initialiseServer which doesn't properly start it up and open a port, (so things are a bit faster) but still allows `server.inject()` to work. https://github.com/DEFRA/btms-portal-frontend/blob/main/test/integration/home.test.js#L3.

Then we can initialise within the test and get rid of the before & after each.